modules/web/js/ballerina/diagram2/views/default/components/nodes/struct-node.jsx

```diff
@@ -187,6 +188,67 @@ class StructNode extends React.Component {
     }
 
     /**
+     * Open JSON import dialog box and pass callback
+     */
+    onClickJsonImport() {
+        const onImport = (json) => {
+            let refExpr = TreeBuilder.build(FragmentUtils.parseFragment(FragmentUtils.createExpressionFragment(json)));
+            let currentValue;
+            this.props.model.setFields([], true);
```

Comment:
Should we check for errors in building the tree. If there is an error, we shouldn't make fields empty.

Reply:
As JSON is validated from ACE editor mode, there should not be errors when parsing